### PR TITLE
title change

### PR DESCRIPTION
**Pull request by zukoaang**
change the title yo.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ class App extends Component {
       },
       {
         id:2,
-        name: "playlist two",
+        name: "Because the Internet",
```
Comment (edenworky) [inline src/App.js]: `Playlist Two` is better. Capitalized, please.

**Review by edenworky — CHANGES_REQUESTED** [review]
Cuz grammar, yo.

File: src/App.js (hunk 1 above)
Comment (zukoaang) [reply src/App.js]: caaaaan doooooo look at mmeeeee

**Review by edenworky — APPROVED** [review]
👏👏👏👏

**Pull request merged**